### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by dave-vazquez**
# Description

Creates a HOC for the Navigation.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [x] Complete, tested, ready to review and merge

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: src/components/Navigation/withNavigation.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  // and the height taken from the nav bars
+  const contentHeight =
+    // if both nav bars are shown
+    displayTop && displayBottom
+      ? "calc(100vh - (100px))"
+      : // if only one nav bar is shown
+        (!displayTop && displayBottom) || (displayTop && !displayBottom)
+        ? "calc(100vh - (50px))"
+        : // if neither nav bar is shown
+          "100vh";
```
Comment (israelgonzalezb) [inline src/components/Navigation/withNavigation.js:27]: this is dank!

**Review by israelgonzalezb — APPROVED** [review]
Looks awesome!

File: src/components/Navigation/withNavigation.js (hunk 1 above)
Comment (dave-vazquez) [reply src/components/Navigation/withNavigation.js:27]: ugh.. i hate typo's in my code.. "defference", but thanks!!

**Review by JimmyMcBride — APPROVED** [review]
Sick! Great work!

**Pull request merged**